Content.Shared/Markings/MarkingsComponent.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+using Content.Shared.CharacterAppearance;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+
+namespace Content.Shared.Markings
+{
+    [RegisterComponent]
```

Comment:
Should it be friend of markingssystem?

Reply:
yeah, but MarkingsSystem is currently clientside only (should this be in client as a result?)